packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -195,6 +195,37 @@ describe('EDiscovery Content API Tests', () => {
 
       return result;
     });
+
+    it('GetContentSummary submits an audit log request', () => {
+      const result = webex.internal.ediscovery.postAuditLog(uuid, {timeoutMs: 1000})
+        .then((res) => {
+          expect(res.statusCode).equal(200);
+          sinon.assert.calledWith(webex.request, sinon.match.has('timeout', 1000));
+        });
+
+      return result;
+    });
+
+    it('GetContentSummary submits a malformed audit request and gets rejected', () => {
+      const noUuid = '';
+      const result = expect(webex.internal.ediscovery.postAuditLog(noUuid, {timeoutMs: 1000})).to.be.rejectedWith(Error, 'Undefined parameter');
```

Comment:
Test is failing because the message on the error message has changed

Reply:
Interesting, i could not get this to fail properly locally, something probably wasn't being rebuilt correctly locally, will update this.